lib/lotus/model/adapters/sql/query.rb

```diff
@@ -663,6 +660,11 @@ def apply(query)
             end
           end
 
+          def _push_to_conditions(condition_type, condition)
+            raise ArgumentError, 'You need to specify a condition.' unless condition
```

Comment:
What do you think about removing this check as Ruby already check for the missing argument